packages/server/.env.development

```diff
@@ -14,7 +14,8 @@ NODE_ENV=development
 GOOGLE_CLIENT_ID=fake_client_id
 GOOGLE_REDIRECT_URI=fake_redirect_uri
 GOOGLE_CLIENT_SECRET=fake_client_secret
-
+MAIL_USER=fake_mail_user
+MAIL_PASS=fake_mail_pass
```

Comment:
This isn't necessarily regarding this PR, but I'm kinda having trouble following in the code, but what are these .env variables used for exactly? Like which parts of the application send emails?

And also, do you think it would be possible to somehow merge them to be the same as @d3li0n `GMAIL_USER` and `GMAIL_PASSWORD`? 

Also wondering your thoughts on this @d3li0n

Reply:
Yeah, it's probably worth having it the same for consistency and merge conflicts reduction

Reply:
removed